### ISSUE TITLE
error opening input file '/Users/franchan/Documents/dev/ios/RefundDelays/Pods/GRDB.swift' (Is a directory)

### ISSUE THREAD

**Issue by aptly-io** [realm/SwiftLint#2035]
I think swxftlint tries to lint GRDB.swift as a source file while in fact it is a directory
See the GRDB project here: https://github.com/groue/GRDB.swift

### New Issue Checklist

- [ x] Updated SwiftLint to the latest version
- [ x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
Linting 'GRDB.swift' (4/566)
error opening input file '/Users/franchan/Documents/dev/ios/RefundDelays/Pods/GRDB.swift' (Is a directory)
Most rules will be skipped because sourcekitd has failed.
Linting 'TableDefinition.swift' (5/566)
Linting 'QueryInterfaceSelectQueryDefinition.swift' (6/566)
Linting 'QueryInterfaceRequest.swift' (7/566)
Fatal error: 'try!' expression unexpectedly raised an error: Error Domain=NSCocoaErrorDomain Code=257 "The file “GRDB.swift” couldn’t be opened because you don’t have permission to view it." UserInfo={NSFilePath=/Users/franchan/Documents/dev/ios/RefundDelays/Pods/GRDB.swift, NSUnderlyingError=0x7f936690ed00 {Error Domain=NSPOSIXErrorDomain Code=13 "Permission denied"}}: file /BuildRoot/Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-900.0.74.1/src/swift/stdlib/public/core/ErrorType.swift, line 181
/Users/franchan/Library/Developer/Xcode/DerivedData/Build/Intermediates.noindex/RefundDelays.build/Debug-iphonesimulator/RefundDelays.build/Script-E7B9E42820250B3700E7FABF.sh: line 6: 48468 Illegal instruction: 4  swiftlint
Command /bin/sh failed with exit code 132
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 0.24.2
* Installation method used (Homebrew, CocoaPods, building from source, etc)? brew
* Paste your configuration file: the default configuration file

```yml
# I use the default configuration that came with brew install for version 0.24.2
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)? No
  If so, paste their relative paths and respective contents.
* Which Xcode version are you using (check `xcode-select -p`)? /Applications/Xcode.app/Contents/Developer
version 9.2
* Do you have a sample that shows the issue? 
No what I did was adding the GRDB framework as a Pod

**Comment by marcelofabri** [comment]
Duplicate of #1948?

**Comment by aptly-io** [comment]
Yes, your right, sorry. It can be closed.
